### DISCUSSION TITLE
GitHub Mobile App Two-factor authentication failed

### DISCUSSION THREAD

**Discussion by mjmagaoay** [community/community#59359] [Mobile]
I tried logging in to my laptop browser and trying to use two-factor via GitHub Mobile. However, even though i typed the correct number, the app says that i typed the wrong number. I tried it many times and still kept on saying that the number typed is wrong 🤦‍♂️ ![Screenshot_20230627_005456_com.github.android.jpg](https://github.com/community/community/assets/23042897/ddbef72a-7cc6-430e-b649-595ef9d41f1a)

![IMG_20230627_005409.jpg](https://github.com/community/community/assets/23042897/f2a69312-4da7-4c93-856f-a5344787fed0)

**Comment by stevepopovich** [comment]
Hi! What version of the app are you using?

**Reply by lijie-lee** [reply]
I encountered this issue two. My app version is 1.199.0(102894307)

**Reply by ItachiSan** [reply]
Same issue, same build. Quite frustrating 🤔

**Reply by Sahnesiguzel33** [reply]
Felaket sinir bozucu

**Comment by ghost** [comment]
59359

**Comment by joshuawalker** [comment]
Hey @magz123! Would you mind [opening a ticket](https://docs.github.com/support/contacting-github-support/creating-a-support-ticket#title-h1) with our Support team so we can look at this more closely with you?

**Comment by luketegel** [comment]
I would try turning off 2FA and turning it back on again, if everything else fails.

**Comment by joshuawalker** [comment]
This issue has been fixed in the next release of GitHub Mobile for Android. If you're continuing to run into this, please [open a ticket with GitHub Support](https://docs.github.com/support/contacting-github-support/creating-a-support-ticket).
